chapter_appendix_math/statistics.md

```diff
@@ -1,99 +1,122 @@
 
 ## Evaluating and Comparing Estimators
 
-In statistics, an *estimator* is a function of given samples for calculating the true parameter $\theta$. We estimate $\theta$ with an estimator $\hat{\theta}_n = \hat{f}(x_1, \ldots, x_n)$ as the best guess of $\theta$ after modeling through the training samples {$x_1, x_2, \ldots, x_n$}. 
+In statistics, an *estimator* is a function of given samples used to estimate the true parameter $\theta$. We will write $\hat{\theta}_n = \hat{f}(x_1, \ldots, x_n)$ for the estimate of $\theta$ after observing the samples {$x_1, x_2, \ldots, x_n$}. 
 
-A better illustration of what we are going to discuss about is plotted in :numref:`fig_comparing_estimators`, where we are estimating the true parameter $\theta$ from the population by the sample estimator $\hat{\theta}_n$.
+We've seen simple examples of estimators before in section :numref:`sec_maximum_likelihood`.  If you have a number of samples from a Bernoulli random variable, then the maximum likelihood estimate for the probability the random variable is one can be obtained by counting the number of ones observed and dividing by the total number of samples.  Similarly, an exercise asked you to show that the maximum likelihood estimate of the mean of a Gaussian given a number of samples is given by the average value of all the samples.  These estimators will almost never give the true value of the parameter, but ideally for a large number of samples the estimate will be close.
 
-![Population parameter and a sample parameter.](../img/comparing_estimators.svg)
-:label:`fig_comparing_estimators`
+As an example, we show below the true density of a Gaussian random variable with mean zero and variance one, along with a collection samples from that Gaussian.  We constructed the $y$ coordinate so every point is visible and the relationship to the original density is clearer.
 
+```{.python .input}
+import d2l
+from mxnet import np, npx
+import random
+from scipy.stats import t
+npx.set_np()
 
-There are a lot functions to model an estimator $\hat{\theta_n}$, such as mean or median. You may be curious about which estimator is the best one? Here, we introduce three most common methods to evaluate and compare estimators: statistical bias, standard deviation, and mean square error. 
+# Sample datapoints and create y coordinate
+epsilon = 0.1
+random.seed(8675309)
+xs = np.random.normal(loc=0, scale=1, size=(300, ))
+ys = [np.sum(np.exp(-(xs[0:i] - xs[i])**2/(2 * epsilon**2)) / 
+      np.sqrt(2*np.pi*epsilon**2))/len(xs) for i in range(len(xs))]
+
+# Compute true density
+xd = np.arange(np.min(xs),np.max(xs),0.01)
+yd = np.exp(-xd**2/2)/np.sqrt(2*np.pi)
+    
+# Plot the results
+d2l.set_figsize()
+d2l.plt.plot(xd,yd)
+d2l.plt.scatter(xs,ys)
+d2l.plt.axvline(x=0)
+d2l.plt.xlabel('x')
```

Comment:
Can we feed xlabel, ylabel etc in d2l.plot():
See https://github.com/d2l-ai/d2l-en/blob/numpy2/d2l/d2l.py#L54

Reply:
Shortened.